packages/crashlytics/e2e/crashlytics.e2e.js

```diff
@@ -91,14 +91,13 @@ describe('crashlytics()', function () {
         // eslint-disable-next-line no-console
         console.warn = msg => {
           // we console.warn for deprecated API, can be removed when we move to v9
-          if (!msg.includes('v8 method is deprecated')) {
+          if (!msg.includes('method is deprecated')) {
```

Comment:
These were the tests failing in CI - of course! They were still looking for 'v8' in the warning message but we've worked to remove that. Obvious in hindsight.